docs/DeveloperGuide.adoc

```diff
@@ -335,8 +335,50 @@ image::UndoRedoActivityDiagram.png[width="650"]
 // end::schedule[]
 
 // tag::addExpenses[]
```

Comment:
Do edit the tag here and at the bottom (will add another comment there) if you are intending to do expenses as a whole